proposals/dao/fip_83.ts

```diff
@@ -0,0 +1,89 @@
+import hre, { ethers, artifacts } from 'hardhat';
+import { expect } from 'chai';
+import {
+  DeployUpgradeFunc,
+  NamedAddresses,
+  SetupUpgradeFunc,
+  TeardownUpgradeFunc,
+  ValidateUpgradeFunc
+} from '@custom-types/types';
+import { getImpersonatedSigner, time } from '@test/helpers';
+import { forceEth } from '@test/integration/setup/utils';
+
+const fipNumber = '83';
+const eswak = '0x6ef71cA9cD708883E129559F5edBFb9d9D5C6148';
+
+// Do any deployments
+// This should exclusively include new contract deployments
+const deploy: DeployUpgradeFunc = async (deployAddress: string, addresses: NamedAddresses, logging: boolean) => {
+  const linearTimelockFactory = await ethers.getContractFactory('LinearTokenTimelock');
+  const laTribuFeiTimelock = await linearTimelockFactory.deploy(
+    eswak,
+    '126144000', // 4 years
+    addresses.fei,
+    '0', // no cliff
+    addresses.feiDAOTimelock,
+    '0'
+  );
+  await laTribuFeiTimelock.deployed();
+  logging && console.log('laTribuFeiTimelock: ', laTribuFeiTimelock.address);
+
+  const quadraticTimelockFactory = await ethers.getContractFactory('QuadraticTokenTimelock');
+  const laTribuTribeTimelock = await quadraticTimelockFactory.deploy(
+    eswak,
```

Comment:
Why are we setting this up so that the beneficiary is `eswak`?

I understand that the plan is then for you to distribute funds to the hired devs, but why don't we first hire the devs and then create timelocks for them where they are the beneficiary?

Reply:
At the moment, this would require `eswak` to distribute funds to the devs. Why not create individual timelocks for the devs at a later date?

Reply:
it's hard to leave a stable situation & accept the hire if you're not even sure the funding is secured on the other side, this is kind of a chicken & egg problem

also I'm likely to make the FEI>EUR conversion myself & just pay them € with a normal contract through my company. If over time they feel confident going fully on-chain, I can change the timelock beneficiary to a smart contract that allows them to claim their respective tokens eventually

for the ~2-3 months where vesting starts but I didn't find anyone yet... it sucks we have to rely on trust but I'm happy to be fired if you see me cheating. These timelocks vest less than my own paycheck anyway & me onboarding new devs to the DAO kinda need a bit of trust anyway

Reply:
Fair enough. I'd prefer a trust minimalised solution but this arrangement seems like it will necessarily require some trust + it has passed snapshot

Reply:
Tagging @ditchfieldcaleb  and @Joeysantoro in case anyone else wants to review

Reply:
Let's trust @eswak to do the right thing here. His rep is way more valuable than this drip.